python/catkin/builder.py

```diff
@@ -702,12 +706,15 @@ def build_package(
 
 
 def get_new_env(package, develspace, installspace, install, last_env, destdir=None):
+    envCommand = 'env.sh'
+    if platform.system() is 'Windows':
+        envCommand = 'env.bat';
```

Comment:
Please rename this variable to `env_script` for a better naming and following the naming style in the file. Also merge the above line into a single line like: `env_script = 'env.%s' % ('bat' if ... else 'sh')`